### ISSUE TITLE
Logging performance on training dataset 

### ISSUE THREAD

**Issue by singhabhinav** [TorchEnsemble-Community/Ensemble-Pytorch#145]
Hi,

I am using voting classifier. Is there anyway to log performance on training set also, so that I can plot training and validation loss and figure out if my model is underfitting or overfitting.

**Comment by xuyxu** [comment]
Maybe you could redirect the print result to a local text file, and parse it to get the training loss. Tensorboard is not used in VotingClassifier since it cannot support parallelization well.
